macros.in

```diff
@@ -1120,6 +1122,23 @@ done \
 %{__patch} %{-p:-p%{-p*}} %{-q:-s}\
 %{__bzr} commit %{-q} -m %{-m*}
 
+# Subversion
+%__scm_setup_svn(q)\
+%{__svnadmin} create .svnrepos\
+%{__svn} mkdir %{-q} -m "Create directory structure." file://`pwd`/.svnrepos/trunk\
+%{__svn} checkout %{-q} file://`pwd`/.svnrepos/trunk ./\
+%{__svn} add %{-q} --force ./\
+%{__svn} commit %{-q} -m "Initial import." && %{__svn} update %{-q}
+
+# Subversion 1.6 doesn't have its own command to apply patches
+#%__scm_apply_svn(qp:m:)\
```

Comment:
It might just be better to take out the stuff for SVN < 1.7, then. The vast majority of RPM based systems that have this macro improvement are unlikely to have older SVN binaries installed systemwide. A check in the autoconf script for SVN 1.7 or newer could probably be sufficient for this.